### ISSUE TITLE
Can't login to RoundCube

### ISSUE THREAD

**Issue by hajzso** [sovereign/sovereign#631]
Fresh install, generated the hash with passlib. I tried logging in with 'username' and 'username@domain', without luck. 

What I did:
- Hashed password with the python passlib code, copied from the tutorial, only replacing 'password' obviously.
- Copied it over to the group_vars/sovereign file, without putting it into double quotes, leaving a space before it, like this; password_hash: hashed_pass  

Questions:
- Do I insert the hashed pass with quotes in the config, or without?
- Do I login with user@domain or simply user? 
- Is it possible to create an user account after installation, without logging in to the admin account first?

**Comment by sigmafeud** [comment]
What error are you getting?

**Comment by hajzso** [comment]
Wrong username/password. I'm certain I am screwing up something either with the password hash generation, or the config file.

**Comment by Titan-C** [comment]
I'm having the same issue, or making the same error. But I can't login for webmail. IMAP access works

**Comment by Titan-C** [comment]
I don't seem to be able to create new users other than the `main_user_name`

**Comment by hajzso** [comment]
I solved this problem, although I don't really have a clear idea of what it was.. I did a reinstall on a fresh VPS and it worked. Some things I noticed:

First time I ran the playbooks, it kept hanging on random tasks, I had to restart it like 5 times until it could finish, and password didn't work. (So maybe if it hangs, and you restart it it scew with the password?)

Next I tried with the hash in double quotes, kept hanging again, and didn't work either;

For the third, successful install I turned off my VPN and started with a fresh VPS again, no double quotes around the hash, and it worked. Be sure to use the correct hashing method though, if you are on macOS it's the passlib one. I don't know if turning off the VPN was what kept it from hanging, but it ran smoothly the first time. If a task hangs on a task for more than 10 minutes, it's almost sure is an error of some kind, even the Diffie Hellman group didn't take more than 7 mins for me. (If you have a slow computer, leave it a bit more obviously)

**Comment by Titan-C** [comment]
It is a mystery when and how sovereign works. I do get to ran the playbooks, my first biggest problem was my vps not having the fuse kernel module for encfs. But once that deactivated it runs well, nevertheless I experienced the problems in https://github.com/sovereign/sovereign/issues/601#issuecomment-255566608 and with a clean installation dropping as many services as possible to the bare minimun(for me IMAP without autoconfig and webmail, and then owncloud) it works. But if I ran all at once they don't work.
